### PR TITLE
Facial Emotion Detection Module Added

### PR DESCRIPTION
**Pull request by gulshan-mittal**
This module has been designed to detect emotions on human faces and this can also combine with Gender Model to predict whether a Man is happy or a woman is happy. The model used for emotion detection is Mini_Xception which is pretty good and provides remarkable results on validation and test set.

@spmallick @vikasguptaiisc Please review the PR.

Thank you.

### CODE REVIEW

**Comment by spmallick** [comment]
@vishwesh5 Please check this out.

**Comment by vishwesh5** [comment]
@gulshan-mittal Your code looks fine. Please rename the folder to `Facial-Emotion-Detection` or `Facial_Emotion_Detection`.

**Comment by vishwesh5** [comment]
@gulshan-mittal The previous directory (`Facial-Emotion_Detection`) has to be removed.

**Comment by vishwesh5** [comment]
Great! @spmallick we can merge the PR. I will update the README once the PR is merged.

**Comment by gulshan-mittal** [comment]
@vishwesh5 Thanks for reviewing.

Now the required changes have been done.

**Comment by vikasguptaiisc** [comment]
I just checked on some other images. The results are decent but the accuracy isn't very impressive. @gulshan-mittal can you do a thorough check with different images and get back on this? We can have a nice blog on this if it is good enough.

P.S : the below image was classified as **happy**
sad

This was classified as **sad**
surprise

Correct results : 
neutral
**Neutral**
smiling
**Happy**

**Comment by spmallick** [comment]
Thanks for the contribution. We are closing this because maintainer testing reported incorrect classifications with no follow-up, the stack is tied to obsolete TensorFlow and Keras versions, and the PR bundles redundant model checkpoints.
